l10n_es_aeat/views/account_journal_view.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!-- Copyright 2022 Moduon - Eduardo de Miguel
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record id="view_account_journal_form" model="ir.ui.view">
+        <field name="model">account.journal</field>
+        <field name="inherit_id" ref="account.view_account_journal_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='type'][last()]" position="after">
+                <field name="thirdparty_invoice" attrs="{'invisible': 1}" />
```

Comment:
```suggestion
                <field name="thirdparty_invoice" invisible="1" />
```

Reply:
Lo pongo de esa manera para que luego al hacer el attrs="{'invisible': [('move_type', 'not in', ('sale', 'purchase'))]}" en cada uno de los módulos, sólo haya que modificar un solo atributo en vez de dos.

Reply:
Bueno, `attrs` tiene prioridad sobre el otro, pero no es crítico este cambio y funciona igual, así que lo dejamos.